packages/service-library/tests/deferred_tasks/test_deferred_tasks.py

```diff
@@ -158,8 +158,20 @@ async def start(self) -> None:
         response = await _tcp_command(
             "init-context",
             {
-                "rabbit": self.rabbit_service.json(**_get_serialization_options()),
-                "redis": self.redis_service.json(**_get_serialization_options()),
+                "rabbit": json_dumps(
+                    model_dump_with_secrets(
+                        self.rabbit_service,
+                        show_secrets=True,
```

Comment:
does it not make sense to put show_secrets inside the serialisation options? could you please double check where this is used, if only in places where you add show_secrets=true then move it to that function